src/utils/institutions-helpers.ts

```diff
@@ -50,6 +60,5 @@ export const sortCustomerInstitutions = <T extends SimpleCustomerInstitution>(cu
 export const getUnitTopLevelCode = (id = '') => {
   const identifier = id.split('/').pop() ?? '';
   const levelIdentifiers = identifier ? identifier.split('.') : [];
-  const topLevel = levelIdentifiers.length > 0 ? levelIdentifiers[0] : null;
-  return topLevel;
+  return levelIdentifiers.length > 0 ? levelIdentifiers[0] : null;
```

Comment:
I keep getting linting error on this unrelated code, so just fixed it.

Reply:
liting i web storm da I assume? Ikke i terminalen (eslint)?